views/RgbLightView/RgbLightView.tsx

```diff
@@ -23,7 +22,12 @@ export const RgbLightView: FC<RgbLightViewProps> = ({ route }) => {
     selectDeviceWithId(state, location, nodeId, deviceId),
   ) as RgbLight;
 
-  const [color, onColorChange] = useInputValue(rgbLight.values.V_RGB);
+  const [color, onColorChange] = useState(rgbLight.values.V_RGB);
+
+  const colorPickerColorChange = (rgbColor: string) => {
+    const colorValue = rgbColor.split('#')[1];
```

Comment:
Why color is with hash? Just return the raw value in the RGB picker component

Reply:
Should for fine